addons/arsenal/functions/fnc_verifyLoadout.sqf

```diff
@@ -70,7 +71,16 @@ private _fnc_filterLoadout = {
 // Loadout might come from a different modpack, which might have different config naming
 _loadout = _loadout call _fnc_filterLoadout;
 
+{
+    private _class = _extendedInfo getOrDefault [_x, ""];
```

Comment:
@LinkIsGrim Voices were previously stored and handled lower case, meaning old loadouts will throw errors when voices are missing.
Do we want to convert the voice to config case only or do something else?

Reply:
Converting _class to configCase seems like the least intrusive option here

Reply:
Agreed, although I presume we'd either exclude insignias or only do it for voice.

Reply:
Actually, couldn't we just use lowercase for voice all-around?

Reply:
No, because when you copy an item classname from the left panel, it would be lower case instead of config case.

Reply:
done